config/types/partition.go

```diff
@@ -15,12 +15,22 @@
 package types
 
 import (
+	"errors"
 	"fmt"
 	"regexp"
 
 	"github.com/coreos/ignition/config/validate/report"
 )
 
+const (
+	uuidRegexStr = "^(|[[:xdigit:]]{8}-[[:xdigit:]]{4}-[[:xdigit:]]{4}-[[:xdigit:]]{4}-[[:xdigit:]]{12})$"
```

Comment:
No clue. The regex was here before, I just moved it: https://github.com/coreos/ignition/blob/master/config/types/partition.go#L42

Think it's incorrect?